Alignment/OfflineValidation/plugins/MuonAlignmentAnalyzer.cc

```diff
@@ -837,7 +837,7 @@ void MuonAlignmentAnalyzer::endJob(){
 
 
 
-        char binLabel[15];
+        char binLabel[40];
 
```

Comment:
GCC error message provides information of min / max sizes for the buffer.

e.g,
```
Alignment/OfflineValidation/plugins/MuonAlignmentAnalyzer.cc:1371:25: note: 'snprintf' output between 18 and 38 bytes into a destination of size 15
```
More here: https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc700/CMSSW_9_0_X_2017-02-10-1100/Alignment/OfflineValidation